docs/core/howto/defer-intro.rst

```diff
@@ -312,161 +408,19 @@ Because if ``f`` raises, ``g`` will be passed a :api:`twisted.python.failure.Fai
 Otherwise, ``g`` will be passed the asynchronous equivalent of the return value of ``f()`` (i.e. ``y``).
 
 
-Inline callbacks - using 'yield'
```

Comment:
You cannot remove this section.   In a PR which deprecates a feature, you cannot delete things (even documentation).  The deprecation and removal policy is described here: https://twistedmatrix.com/documents/current/core/development/policy/compatibility-policy.html


`inlineCallbacks()` have been around for a while, and there are actually a lot of projects and companies which still use them.  Some of these projects/companies may take a while (years) to migrate from `inlineCallbacks()` to the new `await` keyword.

Reply:
Can I put it somewhere else? I don't think it should be in the introduction

Reply:
No